testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -53,30 +59,34 @@ def build_mesh(
         da.to_netcdf(cw_filename)
         plot_cellWidth=True
         if plot_cellWidth:
-            import matplotlib
-            from cartopy import config
-            import cartopy.crs as ccrs
-            matplotlib.use('Agg')
-            fig = plt.figure()
-            fig.set_size_inches(16.0, 8.0)
-            plt.clf()
+            map_name = '3Wbgy5'
+            xmlFile = pkg_resources.resource_filename(
+                __name__, '{}.xml'.format(map_name))
+            _read_xml_colormap(xmlFile, map_name)
+
+            fig = plt.figure(figsize=[16.0, 8.0])
             ax = plt.axes(projection=ccrs.PlateCarree())
             ax.set_global()
-            im = ax.imshow(cellWidth, origin='lower', transform=ccrs.PlateCarree(
-            ), extent=[-180, 180, -90, 90], cmap='jet')
-            ax.coastlines()
+            im = ax.imshow(cellWidth, origin='lower',
+                           transform=ccrs.PlateCarree(),
+                           extent=[-180, 180, -90, 90], cmap='viridis_r',
+                           zorder=0)
+            ax.add_feature(cartopy.feature.LAND, edgecolor='black', zorder=1)
             gl = ax.gridlines(
                 crs=ccrs.PlateCarree(),
                 draw_labels=True,
                 linewidth=1,
                 color='gray',
                 alpha=0.5,
-                linestyle='-')
+                linestyle='-', zorder=2)
             gl.xlabels_top = False
             gl.ylabels_right = False
             plt.title('Grid cell size, km')
             plt.colorbar(im, shrink=.60)
-            plt.savefig('cellWidthGlobal.png')
+            fig.canvas.draw()
+            plt.tight_layout()
+            plt.savefig('cellWidthGlobal.png', bbox_inches='tight')
```

Comment:
`bbox_inches='tight'` eliminates some of the remaining white space around the image so it looks right.

Reply:
Thanks, good to know.